src/components/Pop/Popper.js

```diff
@@ -86,17 +108,23 @@ class Popper extends Component<Props> {
                 }),
                 zIndex,
               }}
-              {...rest}
             >
               <Animate {...animateProps}>
-                {children}
+                <div
+                  className="c-PopPopperContentWrapper"
+                  onMouseLeave={onMouseLeave}
+                  style={{ position: 'relative', zIndex }}
+                >
+                  {this.renderChildren()}
+                </div>
                 {showArrow && (
                   <div className="c-PopPopper__arrowWrapper">
                     <Arrow
                       color={arrowColor}
                       className={arrowClassName}
                       placement={placement}
                       size={arrowSize}
+                      zIndex={zIndex - 1}
```

Comment:
Ensures the base arrow renders behind the Content. This enables the square rotated 45 degrees trick to work.